tests/integration/explain/debug/with_max_join_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package test_explain_debug
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+	explainUtils "github.com/sourcenetwork/defradb/tests/integration/explain"
+)
+
+var debugMaxTypeIndexJoinManyPattern = dataMap{
+	"explain": dataMap{
+		"operationNode": []dataMap{
+			{
+				"selectTopNode": dataMap{
+					"maxNode": dataMap{
+						"selectNode": dataMap{
+							"typeIndexJoin": dataMap{
+								"typeJoinMany": normalTypeJoinPattern,
+							},
+						},
+					},
```

Comment:
question: here `maxNode` is under `selectTopNode` in the `debug` explain but not in the example above ? Why is that?

Reply:
They are testing two different cases. This one is testing an aggregate from within a select and the other is testing an aggregate from the top level.